backend/blueprint/api/helpers/uploads.py

```diff
@@ -0,0 +1,22 @@
+import uuid
+import os
+from flask import current_app as app
+
+
+def generateFileName():
+    return str(uuid.uuid4())
+
+
+def saveToImage(imageFile=None, extension='.png'):
+    imageName = generateFileName() + extension
+    imageDirectory = os.path.join(app.config.get('BASE_DIR'), 'static', 'uploads', 'image')
```

Comment:
one can make a function for this too.
```python
def join(dir, *args, **kwargs):
     return os.path.join(dir, *args, **kwargs)
# ...
def saveToImage(imageFile=None, extension='.png'):
     imageName = generateFileName() + extension
      imageDirectory = join(app.config.get('BASE_DIR'), 'static', 'uploads', 'image')
```

Reply:
@vaibhavsingh97  What kind of a change are you suggesting?
This is a redundant change as what you have proposed is just a wrapper around an inbuilt function that does the exact same.
Now if you consider the `generatefilename` function . It is not redundant as usage of uuid is not readable for a contributor but it defines what it's going to do.
Therefore, It's understandable that he created a function for generating file name but the one you are suggesting is redundant since there's no difference and the original function `os.path.join` is completely readable and self-explanatory. :+1:

Reply:
@vaibhavsingh97 I don't think so that what you're suggesting is correct, since the original function `os.path.join` is self explanatory as to what it will do. 
And your approach of making a wrapper around the inbuilt function that already is self sufficient is something I do not support :sweat_smile:

Reply:
oh! I did not see that @ParthS007 has already explained it in a comment. :+1:

Reply:
ok :+1: